tests/unit/helper/SysExit.php

```diff
@@ -0,0 +1,82 @@
+<?php
```

Comment:
Just noting about the name of this file, as this helper code is meant to be more generic, and not sysexit specific, as it may be used for other purposes later. 

I suggest that the `vipgoci_unittests_indicate_test_id()`, `vipgoci_unittests_check_indication_for_test_id()` and `vipgoci_unittests_remove_indication_for_test_id()` functions is moved to a file named `IndicateTestId.php`or something similar. And suggest that the `vipgoci_unittests_check_irc_api_alert_queue()` function is moved to a file named `CheckIrcApiAlertQueue.php`.